src/spec/Paths.php

```diff
@@ -147,6 +151,18 @@ public function validate(): bool
             if (strpos($key, '/') !== 0) {
                 $this->_errors[] = "Path must begin with /: $key";
             }
+            $operations = $path->getOperations();
+            foreach($operations as $operation) {
+                $id = $operation->operationId;
+                if(empty($id)) {
+                    continue;
+                }
+                if(!empty($this->_operationsIds[$id]) && $this->_operationsIds[$id]) {
+                    $this->_errors[] = "Operation ID '$id', already exist. Operation IDs must be unique.";
+                    $valid = false;
+                }
+                $this->_operationsIds[$id] = true;
```

Comment:
_operationsIds should be a local variable otherwise consecutive calls to validate() will report all ids as duplicate.